tools/fetch-releases.js

```diff
@@ -11,10 +11,20 @@ async function populateReleases() {
     elves.getReleaseInfo(version),
   );
 
-  console.log(`Updating local releases.json with ${releases.length} versions.`);
+  if (releases.length) {
+    console.log(
+      `Updating local releases.json with ${releases.length} versions.`,
+    );
 
-  await fs.remove(file);
-  await fs.outputJSON(file, releases);
+    await fs.remove(file);
+    await fs.outputJSON(file, releases);
```

Comment:
```suggestion
    await fs.outputJSON(file, releases);
```

The `outputJson` function uses the `fs.writeFile` API internally, and the default flag for `fs.writeFile` is `"w"`. This flag means that if the file already exists, it will be overwritten without needing to be deleted first.